spec/requests/api/v2/providers/courses_spec.rb

```diff
@@ -15,7 +15,7 @@
     end
 
     let(:findable_open_course) {
-      create(:course,
+      create(:course, :resulting_in_pgce_with_qts,
```

Comment:
Ain't the default course  
has that?
`resulting_in_pgce_with_qts,`

```ruby 
Factory.define
  factory :course
#those fields
    sequence(:course_code) { |n| "C#{n}D3" }
    name { Faker::ProgrammingLanguage.name }
    qualification { 1 }
    association(:provider)
    resulting_in_pgce_with_qts

    transient do
      age { nil }
end

# maybe that is my justification for traits to be here ;P
```

Reply:
The default course does indeed have that, but since we're asserting on `"qualifications" => %w[qts pgce],` further down, it's better IMHO to have an explicit connection between the input data and the output. It also means if the default qualifications in the course factory change, this spec won't break.